packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoTableHeaderCell/styles.ts

```diff
@@ -1,6 +1,11 @@
 import { css } from '@emotion/react';
 
 import { WfoTheme } from '@/hooks';
+import { SortOrder } from '@/types';
+
+export const HEADER_CELL_TITLE_BUTTON = 'headerCellTitleButton';
```

Comment:
Are these default eui classes? Maybe postfix the const name with _CLASS

Reply:
done